backend/game/cards/keep_card.py

```diff
@@ -0,0 +1,9 @@
+from abc import abstractmethod
+
+from game.cards.card import Card
+
+
+class KeepCard(Card):
+    @abstractmethod
```

Comment:
This abstract method should be placed on the Card class and taken out of the Keep/Discard classes